include/space_tech_20200.h

```diff
@@ -0,0 +1,29 @@
+#include <vector>
+#include <stdio.h>
+
+class SudokuValidator {
+public:
+   bool isValid(const std::vector<std::vector<int>>& grid)
+   {
+       for (int i = 0; i < 9; ++i) {
+           std::vector<bool> rowValues(9, false);
+           std::vector<bool> colValues(9, false);
+
+	   for (int j = 0; j < 9; ++j) {
```

Comment:
This line starts with a tab, and depending on the editor, the line is not shown properly indented. Not very serious issue though.